Gemfile.lock

```diff
@@ -834,7 +834,7 @@ DEPENDENCIES
   zxcvbn (= 0.1.9)
 
 RUBY VERSION
-   ruby 3.2.2p53
+   ruby 3.2.2
```

Comment:
do we need to make sure we as a team are all on the same patch? I would back this change out for now

Reply:
I don't know, but `ruby-build` doesn't seem to offer any patched versions of 3.2.2 unless I'm missing something. I restored it in d863930ae, but I'm going to see this every time...